kubernetes/charts/edge-kubernetes/templates/_helpers.tpl

```diff
@@ -48,10 +48,10 @@ agent:
     StorageClassName: {{- if (eq "-" .Values.edgeAgent.env.storageClassName) }} "" {{- else }} {{ .Values.edgeAgent.env.storageClassName | quote }} {{- end }}
     {{- end }}
     {{- if .Values.edgeAgent.env.enableExperimentalFeatures }}
-    ExperimentalFeatures__Enabled: {{ .Values.edgeAgent.env.enableExperimentalFeatures }}
+    ExperimentalFeatures__Enabled: {{ .Values.edgeAgent.env.enableExperimentalFeature | quotes }}
```

Comment:
I have seen "quote" being used before - singular quote - in helm templates. Is "quotes" (plural) a different processor command?

Reply:
Yeah, seems like I cannot copy paste today.